shell/utils/settings.ts

```diff
@@ -104,5 +104,11 @@ export const getPerformanceSetting = (rootGetters: Record<string, (arg0: string,
   // Start with the default and overwrite the values from the setting - ensures we have defaults for newly added options
   const safeDefaults = Object.assign({}, DEFAULT_PERF_SETTING);
 
-  return Object.assign(safeDefaults, perfSetting || {});
+  const r = Object.assign(safeDefaults, perfSetting || {});
+
+  r.serverPagination.stores.cluster.resources.enableSome.enabled.push('provisioning.cattle.io.cluster');
+  r.serverPagination.stores.cluster.resources.enableSome.enabled.push('cluster');
+  r.serverPagination.stores.cluster.resources.enableSome.enabled.push('rke.cattle.io.etcdsnapshot');
```

Comment:
Temporary to allow paginated requests

Reply:
in theory the three above shouldn't change anything as they're in the `cluster` store rather than `management`.

to get these without code changes, and fix the prov and mgmt cluster contexts (which are now not needed) `shell/config/settings.ts` `serverPagination` can be updated and then the new button `Populate with latest pagination defaults` can be used (and then applied). That process will soon be greatly updated (i have a PR, but it's in flux)